spec/factories.rb

```diff
@@ -231,7 +228,9 @@
     subscriber_list
   end
 
-  factory :user
+  factory :user do
+    uid { SecureRandom.uuid }
```

Comment:
How come this change is here?

Reply:
Due to the updated test [here](https://github.com/alphagov/email-alert-api/pull/1408/files#diff-9066f9057bbc4594aadaf7c321dd4506R19-R20).

After updating the test to a `change` block it started to fail as it was comparing nil (default signon_user_id value for delivery attempts) against nil (user's uid passed in). This seemed like a bad test so I created a uid value in the factory as I assumed all user's should have one.

Reply:
Oh, I see. Thanks for explaining!